src/main/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/CreateCollectionCommandResolver.java

```diff
@@ -20,7 +20,7 @@
 public class CreateCollectionCommandResolver implements CommandResolver<CreateCollectionCommand> {
 
   private final ObjectMapper objectMapper;
-  private final SchemaManager schemaManager;
+  @Inject CQLSessionCache cqlSessionCache;
```

Comment:
Let's remove `@Inject` from here, injected via constructor (won't break anything either way, just cleaner). And make `final`.